admin/css/picker.css

```diff
@@ -313,11 +430,12 @@
   }
 
   .image-picker.open-sidebar {
-    width: calc(100% - var(--sidebar-width-tablet) - 52px);
+    width: 100%;
```

Comment:
Probably because of that change, now when the sidebar is open we no longer see the help tooltip

Reply:
I've just pushed a new change which modify the way of scaling the list. The help tooltip is now visible every time.

Reply:
I still don't see the help tooltip on mobile, and on tablet when the sidebar is open 😕

Reply:
On large screens (not XL) it's also a bit cut off:

<img width="924" alt="Screenshot 2023-08-16 at 13 27 44" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/d45e2493-97cd-45fb-895a-6f2f79ecdd87">

Reply:
It's now a bit off, it should be on top of the last image but instead it's outside of the list

<img width="318" alt="Screenshot 2023-08-18 at 13 41 26" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/1559ff66-bba3-431a-8908-a6f71b68633f">